cdap-ui/app/cdap/components/FieldLevelLineage/v2/Context/FllContext.tsx

```diff
@@ -45,40 +52,56 @@ export interface IContextState {
   causeSets: ITableFields;
   impactSets: ITableFields;
   showingOneField: boolean;
+  start: number | string | null;
```

Comment:
One suggestion. Have the `number | string | null` as a custom type in this file and use it everywhere.

```
type ITimeType = number | string | null;
```